pkg/lib/cockpit-components-table.jsx

```diff
@@ -228,8 +228,10 @@ export const ListingTable = ({
                             }
                         }} />
                     }
-                    {row.columns.map((cell, cellIndex) => {
+                    {row.columns.map((cell, prevCellsCnt) => {
                         const { key, ...cellProps } = cell.props || {};
+                        const prevCells = row.columns.slice(0, prevCellsCnt);
+                        const cellIndex = prevCells.reduce((acc, curr) => acc + (curr.props && curr.props.colSpan ? curr.props.colSpan : 1), 0);
```

Comment:
This added line is not executed by any test.

Reply:
it's covered by anaconda-webui tests.